expect_test.go

```diff
@@ -821,6 +821,51 @@ func TestSpawnGeneric(t *testing.T) {
 	}
 }
 
+// TestSendTimeout tests that Send command can fail on timeout.
+func TestSendTimeout(t *testing.T) {
+	t.Log("Running test: TestSendTimeout")
+	rIn, wIn := io.Pipe()
+	rOut, wOut := io.Pipe()
+	waitCh := make(chan error)
+	outCh := make(chan string)
+	defer close(outCh)
+
+	go fakeCli(cliMap, rIn, wOut)
+	exp, r, err := SpawnGeneric(
+		&GenOptions{
+			In:    wIn,
+			Out:   rOut,
+			Wait:  func() error { return <-waitCh },
+			Close: func() error { return nil },
+			Check: func() bool { return true },
+		}, -1, SendTimeout(10*time.Second))
+	if err != nil {
+		t.Errorf("test: TestSendTimeout, SpawnGeneric failed: %v", err)
```

Comment:
Since this test is not TableDriven the "test:" is not needed. 

Also you don't need the "TestSendTimeout" since it shown anyway .. 

What about something like:

  t.Errorf("SpawnGeneric(_, %d , SendTimeout(%v)) failed: %v", -1, 10*time.Second,err)

Goes for the other errors too.